contrib/kind.sh

```diff
@@ -23,6 +23,10 @@ run_kubectl() {
 MASTER_COUNT=1
 K8S_VERSION=${K8S_VERSION:-v1.17.2}
 KIND_INSTALL_INGRESS=${KIND_INSTALL_INGRESS:-false}
+export OVN_HYBRID_OVERLAY_ENABLE=${OVN_HYBRID_OVERLAY_ENABLE:-true}
+if [ "$OVN_HYBRID_OVERLAY_ENABLE" == false ]; then
```

Comment:
!= true

do we need to unset it? shouldn't rendering take care of bool values?

Reply:
The value is just a set or not set for `OVN_HYBRID_OVERLAY_ENABLE`. For example, `OVN_HYBRID_OVERLAY_ENABLE=1` would enable it when passed to the runtime. So if anything is set here, it will enable HO. I unset it to give the user the option to not enable it. It also presumes HO is enabled by default as you commented on above.

Reply:
If false is set I don't think it should enable HO right?
https://github.com/ovn-org/ovn-kubernetes/blob/master/dist/templates/ovnkube-node.yaml.j2#L240

If it does then that's a bug I think.

Reply:
It is a bug, change this line also in this commit to check for true:
https://github.com/ovn-org/ovn-kubernetes/blob/master/dist/images/ovnkube.sh#L752

Reply:
- Fixed the line in `ovnkube.sh` to behave more like a bool.
- Removed defaulting `kind.sh` to `OVN_HYBRID_OVERLAY_ENABLE=true`
- Added HO to [.github/workflows/test.yml](https://github.com/ovn-org/ovn-kubernetes/pull/1272/files#diff-fe8421955fd596131bb6f1b78984b2fb).